frontend/gpay_agent/lib/agent_datamodels/agent.dart

```diff
@@ -31,4 +31,11 @@ class Agent{
         + 'AgentLatitude: $AgentLatitude, '
         + 'AgentLongitude: $AgentLongitude}';
   }
+
+  String getName() {
+    if (AgentMidName != "")
+      return (AgentFirstName+" "+AgentMidName+" "+AgentLastName);
```

Comment:
please add space before and after operators
This should be handled if you use this
https://dart.dev/guides/language/effective-dart/style#do-format-your-code-using-dartfmt